src/main/kotlin/com/manimdsl/ManimDSLParser.kt

```diff
@@ -18,6 +19,7 @@ enum class ExitStatus(val code: Int) {
     EXIT_SUCCESS(0),
     SYNTAX_ERROR(100),
     SEMANTIC_ERROR(200),
+    RUNTIME_ERROR(300),
```

Comment:
Can we add this to documentation (should just an extra line added)

Reply:
Done